packages/back-end/src/services/organizations.ts

```diff
@@ -245,13 +252,33 @@ export async function revokeInvite(
     invites,
   });
 
-  return organization;
+  const updatedOrganization = cloneDeep(organization);
+  updatedOrganization.invites = invites;
+  await updateSubscriptionIfProLicense(updatedOrganization);
+
+  return updatedOrganization;
 }
 
 export function getInviteUrl(key: string) {
   return `${APP_ORIGIN}/invitation?key=${key}`;
 }
 
+async function updateSubscriptionIfProLicense(
```

Comment:
We have an audit of all changes to the license in the license database.  As the license is the source of truth of the stripe subscriptions now I feel like it makes sense that we keep the records in that database, and the audit collection is probably good enough.  

We now check whether the stripe subscription is during the daily license check and update it if need be.  So it can't really get out of line.

The license database will have both cloud Pro and self hosted pro, so it should be more accurate than any database within growthbook cloud mongo repo as well (once all current licenses in growthbook cloud get moved over)

In terms of the daily Slack bot, perhaps we can instead just have a dashboard somewhere in Mode that has various charts over time for tall the data that the daily aws lambda sync calculates?  I think it should have access to the license database and hopefully can calculate everything it needs from there?  What do you think of that idea?